app/components/views/Bond/index.tsx

```diff
@@ -51,6 +51,7 @@ import * as styles from "./styles";
 import { BondBalancesCard } from "components/BondBalancesCard";
 import { Image } from "components/Image";
 import { ImageCard } from "components/ImageCard";
+import { ButtonBaseProps } from "@klimadao/lib/components/Buttons/ButtonBase";
```

Comment:
```suggestion
import { ButtonBaseProps } from "@klimadao/lib/components";
```

I'm surprised this specific import path works. It probably only works because it's a type and types are disposed of at compile time.
When I set up the repo I was only able to get imports working via the `@klimadao/lib/components` barrel file.
Im obliged to say we stick to that import path, if only for consistency

Reply:
Makes sense, updated